<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View_FunctionGuideWindow</class>
 <widget class="QWidget" name="View_FunctionGuideWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>562</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="previousPagePushButton">
        <property name="text">
         <string>上一页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextPagePushButton">
        <property name="text">
         <string>下一页</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="showFunctionGuideTabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="createWidget">
      <attribute name="title">
       <string>新建</string>
      </attribute>
      <widget class="QLabel" name="showCreatePixmapLabel">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>craetePixmap</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="openSystemFileWidget">
      <attribute name="title">
       <string>打开系统文件</string>
      </attribute>
      <widget class="QLabel" name="showOpenSystemFileLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>openPixmap</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="findOnePathWidget">
      <attribute name="title">
       <string>查找单路径文件</string>
      </attribute>
      <widget class="QLabel" name="showFindOnePathLabel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>findOnePath</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="findAllePathWidget">
      <attribute name="title">
       <string>查找全盘文件</string>
      </attribute>
      <widget class="QLabel" name="showFindAllPathLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>findAllFile</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
